Source/Krypton Components/Directory.Build.props

```diff
@@ -1,44 +1,83 @@
 <Project>
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../../'))" />
-
-  <PropertyGroup>
-    <SignAssembly>false</SignAssembly>
-    <AssemblyOriginatorKeyFile>StrongKrypton.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
-
-  <!--Handle AssemblyInfo-->
-  <PropertyGroup>
-    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
-    <NeutralLanguage>en</NeutralLanguage>
-    <Authors>Phil Wright (A.K.A ComponentFactory), Peter Wagner (A.K.A Wagnerp) and Simon Coghlan (A.K.A Smurf-IV)</Authors>
-    <Copyright>© Component Factory Pty Ltd, 2006 - 2016. Then modifications by Peter Wagner (aka Wagnerp) and Simon Coghlan (aka Smurf-IV) 2017 - 2022. All rights reserved.</Copyright>
-  </PropertyGroup>
-  
-  <PropertyGroup>
-    <!-- common project data -->
-	  <Version>$(LibraryVersion)</Version>
-    <!-- two flavors of TFM are supported : 'all' where the NuGet package include all supported tfms. 'lite' with only .net core and .net 4.5 support -->
-    <TFMs Condition="'$(TFMs)' == ''">lite</TFMs>
-  </PropertyGroup>
-  
-  <PropertyGroup>
-    <!-- common package data -->
-    <PackageProjectUrl>https://github.com/Krypton-Suite/Standard-Toolkit</PackageProjectUrl>
-	  <PackageIcon>Krypton Nightly.png</PackageIcon>
-	  <Authors>Peter William Wagner &amp; Simon Coghlan &amp; Thomas Bolon</Authors>
-	  <PackageLicenseExpression>BSD-3-Clause</PackageLicenseExpression>
-	  <PackageTags>Krypton ComponentFactory WinForms Themes Controls DataGrid Ribbon Workspace Tabs .Net Toolkit Core</PackageTags>
-	  <PackageReleaseNotes>Get updates here: https://github.com/Krypton-Suite/Standard-Toolkit</PackageReleaseNotes>
-  </PropertyGroup>
-  
-  <ItemGroup>
-    <!--Uncomment if your creating Canary packages-->
-    <!--<None Include="../../../Assets/PNG/NuGet Package Icons/Krypton Canary.png" Link="Icon.png" Pack="true" PackagePath="" />-->
-
-    <!--Uncomment if your creating Nightly packages-->
-    <None Include="../../../Assets/PNG/NuGet Package Icons/Krypton Nightly.png" Link="Icon.png" Pack="true" PackagePath="" />
-
-    <!--Uncomment if your creating Stable/Signed packages-->
-    <!--<None Include="../../../Assets/PNG/NuGet Package Icons/Krypton Stable.png" Link="Icon.png" Pack="true" PackagePath="" />-->
-  </ItemGroup>
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../../'))" />
+
+	<PropertyGroup>
+		<SignAssembly>false</SignAssembly>
+		<AssemblyOriginatorKeyFile>StrongKrypton.snk</AssemblyOriginatorKeyFile>
+	</PropertyGroup>
+
+	<!--Handle AssemblyInfo-->
+	<PropertyGroup>
+		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
```

Comment:
Why have the spaces been replaced with tabs ?

Reply:
Might have happened when I was playing about